Steam Web Integration.user.js

```diff
@@ -31,7 +31,7 @@
 // @run-at       document-start
 // @supportURL   https://github.com/Revadike/SteamWebIntegration/issues/
 // @updateURL    https://github.com/Revadike/SteamWebIntegration/raw/master/Steam%20Web%20Integration.user.js
-// @version      1.12.1
+// @version      1.12.2
```

Comment:
version bump not needed for this

Reply:
I dunno what makes ze website deploy, and it *does* (with the JS part living in the script and not the HTML) modify script behaviour.
I'd usually commit changes to this w/o version bump so that they're included whenever you next release anything, but at the same time with commits going to master and master being a potential user update URL, it leads to inconsistent versions being downloaded by users.
(Sorry, not meaning to argue for a dev/trunk/whatever branch here.)